instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -62,7 +62,7 @@ public LoggingEventMapper(
         captureMdcAttributes.size() == 1 && captureMdcAttributes.get(0).equals("*");
   }
 
-  public void emit(LoggerProvider loggerProvider, ILoggingEvent event) {
+  public void emit(LoggerProvider loggerProvider, LoggingEventWithThreadId event) {
```

Comment:
so we only need to pay the cost of the wrapper on the cached startup logs

```suggestion
  public void emit(LoggerProvider loggerProvider, ILoggingEvent event, int threadId) {
```

Reply:
Does not match with "maybe we should just not capture thread.id in logback appender and note the limitation" comment